ktor-server/ktor-server-netty/jvm/src/io/ktor/server/netty/NettyApplicationCall.kt

```diff
@@ -15,18 +16,38 @@ import kotlinx.coroutines.*
 public abstract class NettyApplicationCall(
     application: Application,
     public val context: ChannelHandlerContext,
-    private val requestMessage: Any
+    private val requestMessage: Any,
 ) : BaseApplicationCall(application) {
 
     @OptIn(InternalAPI::class)
     public abstract override val request: NettyApplicationRequest
     @OptIn(InternalAPI::class)
     public abstract override val response: NettyApplicationResponse
+    @OptIn(InternalAPI::class)
+    public lateinit var previousCallFinished: ChannelPromise
+    @OptIn(InternalAPI::class)
+    public lateinit var callFinished: ChannelPromise
 
     public val responseWriteJob: Job = Job()
 
     private val messageReleased = atomic(false)
 
+    internal var isRaw = false
```

Comment:
Please, add a name and/or comment.
What is "raw" here? 
NettyApplicationCall is raw? What is a raw application call?

Reply:
yep, I would rename

Reply:
Thanks for the reviews!
Renamed to `isByteBufferContent`